src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -12,18 +12,26 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
-from full_activity_dump_helper import getFullActivityDumpSasTestHarness, getFullActivityDumpSasUut
 import json
+import time
 import os
+from reference_models.common import mpool
+from functools import partial
 import sas
 import sas_testcase
-from sas_test_harness import SasTestHarnessServer, generateCbsdRecords, \
-    generatePpaRecords
-import time
 import test_harness_objects
+from full_activity_dump import FullActivityDump
+from full_activity_dump_helper import getFullActivityDumpSasTestHarness, getFullActivityDumpSasUut
 from util import winnforum_testcase, configurable_testcase, getCertificateFingerprint, writeConfig, \
-  loadConfig, makePpaAndPalRecordsConsistent
+        loadConfig, makePpaAndPalRecordsConsistent
+from sas_test_harness import SasTestHarnessServer, generateCbsdRecords, \
+        generatePpaRecords
+from reference_models.pre_iap_filtering import pre_iap_filtering
+from reference_models.iap import iap
+from reference_models.interference import aggregate_interference, interference
+
 
+IAP_MARGIN = 1 # Threshold value in dBm
```

Comment:
Should this be DELTA_IAP? If not, I'm not sure by the name what it does.

Reply:
Changed the variable 